packages/nhost_graphql_adapter/example/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ environment:
   sdk: ">=2.18.0 <4.0.0"
 
 dependencies:
-  graphql: ^5.1.3
```

Comment:
please, let's not upgrade to beta versions unless there is a very good reason for it